App.js

```diff
@@ -23,13 +23,25 @@ const App = () => {
     <NavigationContainer>
       <Stack.Navigator>
```

Comment:
Podes adicionar **screenOptions** que vão ser aplicadas a todos os screens
```suggestion
      <Stack.Navigator screenOptions={{ headerShown: false }}>
```